app/components/mandaat/publicatie-status-pill.hbs

```diff
@@ -3,7 +3,11 @@
     @skin={{if
       (eq @mandataris.publicationStatus.label "Bekrachtigd")
       "success"
-      "default"
+      (if
```

Comment:
i was going to rework this like in the mandataris-status-pill BUT ran into the fact that the publication status is not always loaded and didn't want to deal with promise hell